events/smart-descriptions.json

```diff
@@ -1,4 +1,337 @@
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "network.protocol",
+                    "value": "ESMTP"
+                }
+            ],
+            "relationships": [],
+            "value": "{email.from} sent {file.size} bits over protocol {network.protocol} and used the following relay {action.properties.relay}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "action.properties.mailer",
+                    "value": "esmtp"
+                }
+            ],
+            "relationships": [],
+            "value": "An email from relay: {destination.domain} was sent to {user.email} with status: {event.reason}"
+        },{
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "event.category",
+                    "value": "smtp"
+                },
+                {
+                    "field": "email.to",
+                    "value": "postmaster"
+                },
+                {
+                    "field": "action.properties.mailer",
+                    "value": "local"
+                }
+            ],
+            "relationships": [],
+            "value": "A {action.properties.mailer} message to {email.to} with status message {event.reason}"
+        }, {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "action.properties.start_tls",
+                    "value": "client"
+                },
+                {
+                    "field": "event.reason",
+                    "value": "unable to get local issuer certificate"
+                }
+            ],
+            "relationships": [],
+            "value": "The {action.properties.start_tls} is {event.reason}"
+        }, {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "action.properties.start_tls",
+                    "value": "server"
+                },
+                {
+                    "field": "event.reason",
+                    "value": "unable to get local issuer certificate"
+                }
+            ],
+            "relationships": [],
+            "value": "The {action.properties.start_tls} is {event.reason}"
```

Comment:
Maybe this could be a single description by removing the `action.properties.start_tls` condition ?